src/common.rs

```diff
@@ -355,6 +355,60 @@ pub struct LoadAvg {
     pub fifteen: f64,
 }
 
+macro_rules! xid {
+    ($(#[$outer:meta])* $name:ident, $type:ty) => {
```

Comment:
```suggestion
    ($(#[$outer:meta])+ $name:ident, $type:ty) => {
```

To enforce that a doc comment is given in any case.